app/views/errors/unauthorised.html.haml

```diff
@@ -1,4 +1,9 @@
-- content_for :page_title_prefix, t('error_pages.unauthorised')
+- content_for :page_title_prefix, t('error_pages.unauthorised.title')
 
-%h1.govuk-heading-xl=t('error_pages.unauthorised')
+%h1.govuk-heading-xl=t('error_pages.unauthorised.title')
+%p= t('error_pages.unauthorised.text_html')
+%ul.govuk-list.govuk-list--bullet
+  -t('error_pages.unauthorised.options').each do |li|
+    %li=li.html_safe
```

Comment:
I think this should work without the `.html_safe`

Reply:
no because one option has html in it, and i thought this was the least hacky way to do it